amr-wind/wind_energy/actuator/disk/uniform_ct_ops.cpp

```diff
@@ -192,7 +192,7 @@ void optional_parameters(UniformCt::MetaType& meta, const utils::ActParser& pp)
     if (pp.contains("yaw")) {
         amrex::Real yaw;
         pp.get("yaw", yaw);
-        normalRotOp = normalRotOp & vs::zrot(yaw);
+        normalRotOp = normalRotOp & vs::zrot(-yaw);
```

Comment:
Oh, this isn't a critical point, but I would add comments here that indicate the `-yaw` is so that the rotation is applied in a clockwise direction starting from north = zero degrees.  Mostly so that future me can remember how the conventions worked and prevent additional confusion.

Reply:
yeah that's a good point. I'll add that